.github/workflows/set-version.yml

```diff
@@ -0,0 +1,54 @@
+name: Set Next Version in SDK and Swazzler
+
+on:
+  workflow_call:
+    inputs:
+      version:
+        required: true
+        type: string
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'New version, Specify <major.minor>, e.g. 6.4 (Do NOT include -SNAPSHOT, will be added automatically)'
+        required: true
+
+jobs:
+  set-version:
+    name: Set Version in SDK
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    steps:
+      - name: Configure git
+        run: |
+          git config --global user.name 'embrace-ci[bot]'
+          git config --global user.email 'embrace-ci@users.noreply.github.com'
+          git config --global url."https://${{ secrets.CD_GITHUB_USER }}:${{ secrets.CD_GITHUB_TOKEN }}@github.com".insteadOf "https://github.com"
```

Comment:
This should be removed because we no longer authenticate with HTTP now. The token set in the next step should be good enough